### PR TITLE
[MV-417] Allow to turn on webrtc logs in mobile apps

### PR DESCRIPTION
**Pull request by skyman503**

### CODE REVIEW

File: package-lock.json
Comment (graszka22) [inline package-lock.json:1]: ?

File: MembraneRTC/Package.swift [hunk 1]
```diff
@@ -16,7 +16,7 @@ let package = Package(
         )
     ],
     dependencies: [
-        .package(name: "WebRTC", url: "https://github.com/webrtc-sdk/Specs.git", .exact("104.5112.14")),
+        .package(name: "WebRTC", url: "https://github.com/webrtc-sdk/Specs.git", .exact("104.5112.15")),
```
Comment (graszka22) [inline MembraneRTC/Package.swift:19]: please bump the version also in `.podspec` file (see here: https://github.com/jellyfish-dev/membrane-webrtc-ios/commit/b8749af1218b8be754e65781035a9d66b0ea46cf)

The reason why we have the version specified in two places is that swift has two package managers (swift-package and pods) and we use both of them. Pods are needed for react-native and swift-package was there before and I've never really thought about removing it, someone may use it in projects that use swift-package and it's convenient for ios developing

**Review by graszka22 — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (skyman503) [reply package-lock.json:1]: when I was dealing with the checksum issue I tried updating packages and I guess that's what was left of it. Removed now

**Review by graszka22 — APPROVED** [review]
Let's merge after CI passes 👍

File: MembraneRTC/Package.swift (hunk 1 above)
Comment (skyman503) [reply MembraneRTC/Package.swift:19]: I added dependency in the `.podspec`. However I find it super counterintuitive to use simultaneously two package managers. And mby we should think about it if we were to have monorepo for sdks

**Pull request merged**